content/termine.ics/contents+en.lr

```diff
@@ -639,3 +636,7 @@ ort: Campus of the Überlingen Waldorf School
 url: https://ffbsee.net/camp/
 ----
 description: The Technology Camp 2020 will take place in the middle of August in the Lake Constance region near Überlingen! The Technology Camp is a small international meeting of hackers, technology enthusiasts and interested citizens. For four days there will be exchange and discussion, but also programming or tinkering. There will be a lecture program and workshops, as well as joint activities.
+----
+confirmed: yes
+---
+title: Termine Freifunk Bodensee
```

Comment:
Ich weiß nicht ob das überhaupt irgendwo sichtbar ist außer in dem Admin, aber müsste der Titel nicht in englisch sein?